src/Widgets/TaskRow.vala

```diff
@@ -19,12 +19,19 @@
 */
 
 public class Tasks.TaskRow : Gtk.ListBoxRow {
+
+    public enum EditMode {
```

Comment:
Does this really need to be an enum? It seems like a bool would cover it

Reply:
Enum is not mandatory, just thought it’s more expressive than a bool.

My personal favorite would be to detect this state based on the ECal.Component itself (some kind of is_new logic). But I did not figure out how.